app/main.py

```diff
@@ -101,7 +101,9 @@ async def handle_login(request: Request):
         gamemode=0,
     )
 
-    response_data = packets.write_login_reply_packet(presence["user_id"])
+    response_data = bytearray()
+    response_data += packets.write_protocol_version_packet(19)
```

Comment:
sorry i should have realized this before - the first thing we need to do is establish an agreement of which version of the protocol we'd like to use to communicate (between client & server)

Reply:
version 19 is osu's latest protocol version and the one which the stable/beta/cuttingedge clients use